includes/admin/class-wcmp-export.php

```diff
@@ -1595,14 +1596,25 @@ public function generateCustomsDeclaration(WC_Order $wcOrder): CustomsDeclaratio
             ->setWeight($totalWeight);
 
         foreach ($wcOrder->get_items() as $item) {
-            $product       = $item->get_product();
-            $productHelper = new ExportRow($wcOrder, $product);
+            $product = $item->get_product();
+            if (! $product) {
+                Messages::showAdminNotice(
+                    sprintf(
+                        __('warning_product_missing_check_backoffice', 'woocommerce-myparcel'),
+                        $wcOrder->get_id()
+                    ),
+                    Messages::NOTICE_LEVEL_WARNING
+                );
+                $product = new WC_Product();
+                $product->set_weight(1);
+            }
 
             if (! $product || $product->is_virtual()) {
                 continue;
             }
 
-            $customsItem = (new MyParcelCustomsItem())
+            $productHelper = new ExportRow($wcOrder, $product);
```

Comment:
Ik schrik er echt van om "ExportRow" te zien na het voorval in de SDK 😂

Reply:
:-P